homeassistant/helpers/llm.py

```diff
@@ -469,3 +473,190 @@ def _get_exposed_entities(
         entities[state.entity_id] = info
 
     return entities
+
+
+def selector_serializer(schema: Any) -> Any:  # noqa: C901
+    """Convert selectors into OpenAPI schema."""
+    if not isinstance(schema, selector.Selector):
+        return UNSUPPORTED
+
+    if isinstance(schema, selector.BackupLocationSelector):
+        return {"type": "string", "pattern": "^(?:\\/backup|\\w+)$"}
+
+    if isinstance(schema, selector.BooleanSelector):
+        return {"type": "boolean"}
+
+    if isinstance(schema, selector.ColorRGBSelector):
+        return {
+            "type": "array",
+            "items": {"type": "number"},
+            "minItems": 3,
+            "maxItems": 3,
+            "format": "RGB",
+        }
+
+    if isinstance(schema, selector.ConditionSelector):
+        return convert(cv.CONDITIONS_SCHEMA)
+
+    if isinstance(schema, selector.ConstantSelector):
+        return {"enum": [schema.config["value"]]}
+
+    if isinstance(schema, (selector.NumberSelector, selector.ColorTempSelector)):
+        result: dict[str, Any] = {"type": "number"}
+        if "min" in schema.config or "min_mireds" in schema.config:
+            result["minimum"] = schema.config.get("min") or schema.config.get(
```

Comment:
If `min` is 0, it will get `min_mireds`, which might be `None`

Reply:
You are right, but it is consistent with current `ColorTempSelector` behavior: https://github.com/home-assistant/core/blob/fa4b7f307878c20c6530676b672dab4cffea62e9/homeassistant/helpers/selector.py#L430-L438
I will fix both.